example-jobs/envTestJob.groovy

```diff
@@ -2,7 +2,7 @@
 import static jenkins.automation.utils.EnvironmentUtils.isDev
 
 
-def env=${ENVIRONMENT}
+def env="${ENVIRONMENT}"
 if (isDev(env)){
```

Comment:
Because $ENVIRONMENT is only available inside job scripts; it's not accessible directly inside of jenkins-automation classes.

The way around that is to make EnvironmentUtils a proper class with a constructor that takes a string, and stores that string as state, and then turns those existing static methods into nonstatic class methods that simply read the internal state:
psuedo:

```
def env = ""
def EnvironmentUtils(String environment) {
  env = environment
}

def getEnv() {
  return env.ToUpperCase()
}
def isDev() {
  return getEnv() == "DEV"
}

def isProd(){
  return getEnv() == "PROD"
}
```

and then in scripts:

```
envUtil = new EnvironmentUtils("${ENVIRONMENT}")

...

if( envUtil.isDev()){
...
}
```

just a thought.